.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
I am :-1: 
@ChrisBarker-NOAA is +0.5

We need more votes!  @rsignell-usgs and @cbcunc?